libs/blocks/marquee/marquee.css

```diff
@@ -356,13 +361,12 @@
     color: var(--link-color);
   }
 
-  .marquee.tablet-light a:not(.con-button):hover {
-    color: var(--link-hover-color);
+  .marquee.tablet-dark a:is(:not(.con-button), :not(.con-button):hover) {
```

Comment:
same as above for me, not sure i like the `:is(:not(...))` pattern, but understand why.